lib/analysis_options.flutter.yaml

```diff
@@ -0,0 +1,20 @@
+include: package:flutter_lints/flutter.yaml
+
+linter:
+  # Note: Ensure rules are replicated where applicable in `analysis_options.yaml`
+  rules:
+    - avoid_void_async
+    - await_only_futures
+    - curly_braces_in_flow_control_structures
+    - omit_local_variable_types
+    - prefer_single_quotes
+    - require_trailing_commas
+    - unawaited_futures
+    - unnecessary_final
+
+analyzer:
+  # Note: Ensure excludes are replicated in `analysis_options.yaml`
+  exclude:
+    - '**/*.g.dart'
+    - '**/*.reflectable.dart'
+    - '**/__generated__/**'
```

Comment:
Rules take from `jcli git-pr`

Reply:
is a superset of what I had in device_tracking. so i'm happy